polaris-react/src/components/Frame/Frame.tsx

```diff
@@ -73,7 +73,9 @@ interface State {
   showContextualSaveBar: boolean;
 }
 
+const APP_FRAME = 'AppFrame';
 const APP_FRAME_MAIN = 'AppFrameMain';
+const APP_FRAME_CONTENT = 'AppFrameContent';
```

Comment:
Do you think I should export these CONSTs or its ok to copy it into other files?